another-import-service/import-file-parser.js

```diff
@@ -31,7 +31,23 @@ export const importSecondFileParser = async event => {
     for (const record of event.Records) {
         console.log(record.s3.object.key);
 
-        const data = await collectDataForLogging(record.s3.object.key);
+        const data = await collectDataForLogging(record.s3.object.key),
+            sqs = new AWS.SQS();
+
+
+        for (const product of data) {
+            try {
+                const res = await sqs.sendMessage({
+                    QueueUrl: 'https://sqs.eu-west-1.amazonaws.com/517465834650/catalogItemsQueue',
```

Comment:
This should not be hardcoded! And should be received from CloudFormation automatically ;) with a correct serverless config ofc.

Reply:
Example: 

```
- Effect: Allow
          Action:
              - 'sns:*'
          Resource:
              - Fn::ImportValue: SNSTopic
```

where SNSTopic ->

```
SNSTopic:
            Type: AWS::SNS::Topic
            Properties:
                TopicName: createProductTopic
```

and 

```
environment:
        SQS_URL:
            Ref: SQSQueue
```